src/web/components/Input/RadioInput.scss

```diff
@@ -35,4 +36,9 @@
 .radio-option {
   display: flex;
   align-items: center;
+
+  .disabled {
+    background-color: #ebebe4 !important;
```

Comment:
We have `--theme-disabled-button` already - perhaps we could reuse that? Also the `!important` doesn't seem necessary when I tested locally with:
```
background-color: var(--theme-disabled-button);
```

Reply:
I changed it.  Not sure why but the default button background was taking priority when I tested this before.  I also removed the other !important